packages/primitives/package.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "@ndla/primitives",
```

Comment:
Lurer på om vi kanskje skal være frekke nok til å kalle denne `@ndla/select`, `@ndla/dropdown` eller `@ndla/combobox`. Litt redd for at vi har splittet ut for mange primitives i individuelle pakker til at det gir mening å opprette en primitives-pakke.